pkg/onchain/website/uploader.go

```diff
@@ -86,14 +86,20 @@ func upload(client *node.Client, addr []byte, chunks []string, wallet *wallet.Wa
 
 	for index := 0; index < len(chunks); index++ {
 		// Chunk ID encoding
-		params := encodeUint64ToUTF16String(uint64(index))
+		params := convert.U64NbBytes(uint64(index))
 		// Chunk data length encoding
-		params += encodeUint32ToUTF16String(uint32(len(chunks[index])))
+		//nolint:gomnd
+		b := make([]byte, 4)
+		binary.LittleEndian.PutUint32(b, uint32(len(chunks[index])))
+
+		//nolint:ineffassign,nolintlint
+		params = append(params, b...)
 		// Chunk data encoding
-		params += chunks[index]
+		//nolint:ineffassign,nolintlint
+		params = append(params, chunks[index]...)
 
 		//nolint:lll
-		opID, err = onchain.CallFunctionUnwaited(client, *wallet, baseOffset+uint64(index)*multiplicator, addr, "appendBytesToWebsite", []byte(params))
+		opID, err = onchain.CallFunctionUnwaited(client, *wallet, baseOffset+uint64(index)*multiplicator, addr, "appendBytesToWebsite", params)
```

Comment:
why do we have to pass the wallet as a pointer ?
you should use an explaingin varaible because I don't understand the baseOffset+uint64(index)*multiplicator.
And what is multiplicator by the way ?